lib/plugins/aws/package/lib/generateCoreTemplate.js

```diff
@@ -60,12 +60,7 @@ module.exports = {
         });
     }
 
-    if (this.serverless.service.provider.deploymentBucket) {
```

Comment:
This branch was never executed because previous `if` checks exactly same condition and makes an early return